src/rest-server/deploy/stop.sh

```diff
@@ -27,7 +27,7 @@ if kubectl get configmap | grep -q "auth-configuration"; then
     kubectl delete configmap auth-configuration || exit $?
 fi
 
-if kubectl get configmap | grep -q "job-exit-spec-configuration"; then
+if kubectl get configmap | grep -q " job-exit-spec-configuration"; then
```

Comment:
why add a space here?

Reply:
k8s-job-exit-spec-configuration will detected by this grep command